docs/content/preview/migrate/yb-voyager/migrate-steps.md

```diff
@@ -0,0 +1,396 @@
+        --target-db-name dbname \
+        --target-db-schema schemaName
+``` -->
+
+yb-voyager splits the data dump files (from the `$EXPORT_DIR/data` directory) into smaller _batches_ , each of which contains at most `--batch-size` number of records. By default, the `--batch-size` is 100,000 records. yb-voyager concurrently ingests the batches such that all nodes of the target YugabyteDB cluster are used. This phase is designed to be _restartable_ if yb-voyager terminates while the data import is in progress. After restarting, the data import resumes from its current state.
+
+By default, the `yb-voyager import data` command creates one database connection to each of the nodes of the target YugabyteDB cluster. You can increase the number of connections by specifying the total connection count, using the `--parallel-jobs` argument with the `import data` command. The command distributes the connections equally to all the nodes of the cluster.
+
+```sh
+yb-voyager import data --export-dir ${EXPORT_DIR} \
+        --target-db-host ${TARGET_DB_HOST} \
+        --target-db-user ${TARGET_DB_USER} \
+        --target-db-password ${TARGET_DB_PASSWORD} \
+        --target-db-name ${TARGET_DB_NAME} \
+        --target-db-schema ${TARGET_DB_SCHEMA} \
+        --parallel-jobs 100 \
+        --batch-size 250000
+```
+
+{{< tip title="Importing large datasets" >}}
+
+When importing a very large database, run the import data command in a `screen` session, so that the import is not terminated when the terminal session stops.
+
+If the `yb-voyager import data` command terminates before completing the data ingestion, you can re-run it with the same arguments and the command will resume the data import operation.
+
+{{< /tip >}}
+
+#### Import data status
+
+Run the `yb-voyager import data status --export-dir ${EXPORT_DIR}` command to get an overall progress of the data import operation.
+
+#### Import data file
+
+If all your data files are in csv format and you have already created a schema in your target YugabyteDB, you can use the `yb-voyager import data file` command to load the data into the target table directly from the csv file(s). This command doesn’t require performing other migration steps ([export and analyze schema](#export-and-analyze-schema), [export data](#export-data), or [import schema](#import-schema) prior to import. It only requires a table present in the target database to perform the import.
+
+<!-- ```sh
+yb-voyager import data file --export-dir ${EXPORT_DIR} \
+        --target-db-host ${TARGET_DB_HOST} \
+        --target-db-port ${TARGET_DB_PORT} \
+        --target-db-user ${TARGET_DB_USER} \
+        --target-db-password ${TARGET_DB_PASSWORD:-''} \
+        --target-db-name ${TARGET_DB_NAME} \
+        –-data-dir “/path/to/files/dir/” \
+        --file-table-map “filename1:table1,filename2:table2” \
+        --delimiter “|” \
+        –-has-header \
+``` -->
+
+```sh
+yb-voyager import data file--export-dir ${EXPORT_DIR} \
```

Comment:
cc @sanyamsinghal

Reply:
@ddhodge `--export-dir` is required, actually we do some bookkeeping about the state of migration to provide functionality like Resumability in case of any failure and also for faster loading by creating intermediate smaller splits of the original file before loading.